content/en/docs/contribute/review/for-approvers.md

```diff
@@ -79,7 +79,7 @@ The most common prow commands reviewers and approvers use are:
 {{< table caption="Prow commands for reviewing" >}}
 Prow Command | Role Restrictions | Description
 :------------|:------------------|:-----------
-`/lgtm` | Anyone, but triggers automation if a Reviewer or Approver uses it | Signals that you've finished reviewing a PR and are satisfied with the changes.
+`/lgtm` | Organization members | Signals that you've finished reviewing a PR and are satisfied with the changes.
```

Comment:
The existing documentation is correct. Anyone in the Kubernetes organization can mark a PR as LGTM.

Reply:
Oops, I had that backwards.

Two things.
1. This makes sense
2. I will suggest incorporating this into the [contributor site](https://k8s.dev/) and then hyperlinking there.